packages/node_modules/@webex/internal-plugin-conversation/src/encryption-transforms.js

```diff
@@ -159,27 +166,25 @@ export const transforms = toArray('outbound', {
         key = get(activity, 'target.defaultActivityEncryptionKeyUrl');
       }
 
-      if (key) {
```

Comment:
question: this section used to be guarded by an `if (key)`. Why did that need to be removed? Is there a danger for normal conversations now that this code will run when it wasn't meant to?

Reply:
Good catch I didn't want to delete it.